Makefile

```diff
@@ -13,7 +13,7 @@ STORYBOOK_DIR := view/storybook
 DOCS_DIR := view/docs
 WEB_DIR := view/web
 THEME_GENERATOR_DIR := tools/theme-generator
```

Comment:
I think we also moved this to another folder, or changed the folder name to `themegen`